httpbin/core.py

```diff
@@ -907,13 +907,13 @@ def bearer_auth():
       401:
         description: Unsuccessful authentication.
     """
-    if 'Authorization' not in request.headers:
+    authorization = request.headers.get('Authorization')
+    if not (authorization and authorization.startswith('Bearer ')):
         response = app.make_response('')
         response.headers['WWW-Authenticate'] = 'Bearer'
         response.status_code = 401
         return response
-    authorization = request.headers.get('Authorization')
-    token = authorization.lstrip('Bearer ')
+    token = authorization.split('Bearer ', 1).pop()
```

Comment:
Personally I'd rather do:

```py
slice_start = len('Bearer ')
token = authorization[slice_start:]
```

Reply:
To be clear, both options work. I just think this is a little too clever and would rather it be less clever.

Reply:
Done.